jboss/container/wildfly/launch-config/config/added/launch/launch.sh

```diff
@@ -134,7 +134,7 @@ function launchServer() {
   handleExtensions
   if [ $? -ne 0 ]; then
     log_info "Restarting the server"
-    ${cmd} &
+    ${cmd} ${JAVA_PROXY_OPTIONS} ${JBOSS_HA_ARGS} ${JBOSS_MESSAGING_ARGS} &
```

Comment:
${CLI_EXECUTION_OPTS} were deliberately avoided? 

It is used in starting the server on line 129. Looking into what is in ${CLI_EXECUTION_OPTS} that was probably deliberately. Just to make sure.

Reply:
Yes, it was deliberate, to not have the server to re-apply any CLI scripts.

Reply:
Thanks for confirming.